tests/test_component.py

```diff
@@ -35,3 +36,36 @@ def setUp(self):
         }
         self.builder_i18n_nl = Builder(self.builder_json, language='nl', i18n=i18n)
         self.form_i18n_nl = Form(self.form_json, self.builder_i18n_nl)
+
+    def test_schema_dict(self):
```

Comment:
Why is this function here? :) Maybe I miss the point...
Isn't the regular `setUp` not sufficient, or you could change the `setUp`?

Reply:
yes here I'm testing [this](https://github.com/novacode-nl/python-formio-data/pull/3/commits/144d8ceb0faf08530953600ef2fb81bd49f9a0b1#diff-a675be4c2b88076a10d3b7095558e3113902538814202f6953a6bd56710752faR19).

Reply:
Excellent, thanks!